docs/standard/simd.md

```diff
@@ -80,7 +80,7 @@ double[] SimdVectorProd(double[] left, double[] right)
     var offset = Vector<double>.Count;
     double[] result = new double[left.Length];
     int i = 0;
-    for (i = 0; i < left.Length; i += offset)
+    for (i = 0; i + offset <= left.Length; i += offset)
```

Comment:
```suggestion
    for (; i <= left.Length - offset; i += offset)
```
* `i` is already initialized to `0`
* `left.Length - offset` can be kept in a register, and must not be re-evaluated on each iteration (so better performance)

Although I wouldn't use `offset`, rather inline `Vector<double>.Count` or name this variable `vectorSize` or that like. `offset` is quite misleading.
(I see this was there before this PR, but could fix this at the same time.)

Reply:
@gfoidl Fixed, thanks.